ansible/roles/chaos/templates/burnio.j2

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Script for BurnIO Chaos
+#
+# based on https://github.com/Netflix/SimianArmy/blob/master/src/main/resources/scripts/burnio
+# see also https://github.com/Netflix/SimianArmy/wiki/The-Chaos-Monkey-Army#burn-io-simius-occupatus
+
+BURNFILE="{{ burnio_dir }}/burnio"
+
+cat << EOF > {{ chaos_dir }}/burnioloop
+#!/bin/bash
+# do burnio chaos
+while true;
+do
+    dd if=/dev/urandom of=$BURNFILE bs=1M count=128 iflag=fullblock
+    sleep {{ burnio_wait }}
+done
+EOF
```

Comment:
The start script looks like it is blocking. Does this mean you'd need to open up a second shell in order to stop it?

Reply:
Thanks. I missed the nohup command to put it in the background. Original question was if this would block the ansible playboook (so that the playbook never exited). Looks like that isn't a problem